FrameworkSystem/private/standardLogging/LoggerWrapper.py

```diff
@@ -0,0 +1,208 @@
+"""
+Logger wrapper for DIRAC use
+"""
+
+__RCSID__ = "$Id$"
+
+import logging
+
+from DIRAC.FrameworkSystem.private.standardLogging.LoggingWrapper import LoggingWrapper
+from DIRAC.FrameworkSystem.private.standardLogging.LogLevels import LogLevels
+
+
+class LoggerWrapper(object):
+  """
+  Wrapper of the old Logger object:
+  - made to replace transparently gLogger
+  - based on the standard library 'logging'
+  """
+
+  __gLogging = LoggingWrapper()
+
+  def __init__(self, fathername='', name=''):
+    if fathername == '':
+      self.__logger = logging.getLogger(name)
+    else: 
+      self.__logger = logging.getLogger(fathername).getChild(name)
+
+  def initialized(self):
+    logging.warn("Initialized: Deleted method.")
```

Comment:
Yes, I changed all deprecated methods in verbose with the message "Deleted method. Do not use it". Is it fine ?